pkgs/tools/package-management/nix-du/default.nix

```diff
@@ -40,5 +40,6 @@ rustPlatform.buildRustPackage rec {
     license = licenses.lgpl3Only;
     maintainers = [ maintainers.symphorien ];
     platforms = platforms.unix;
+    broken = stdenv.isDarwin;
```

Comment:
Maybe we should set platforms to linux only. I don't think there is any darwin support.

Reply:
But it works fine on `master` and also after this commit it also works fine on Darwin: https://github.com/NixOS/nixpkgs/pull/144197#issuecomment-958168855

Reply:
nix-du works for me on darwin :) (0.3.4 from nixpkgs 2182da1de8081da92f3c5c04e3fa5e5452266afc)

Reply:
If it works then please drop this again.

Reply:
It doesn't work right now because of nix 2.4 bump, but we can update it later on when `nix-du` is updated with the fix.

Reply:
Please add a comment mentioning 2.4 and referencing this PR.